guides/14_Content_Security_Policy.md

```diff
@@ -0,0 +1,51 @@
+# Content Security Policy for applications using the Amazon Chime SDK
+
+Modern web applications use [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) to protect users from certain classes of attacks. You can add a security policy by configuring your web server to return the Content-Security-Policy HTTP header or using `<meta>` element to configure a policy. You can add security headers using [Lambda@Edge](https://docs.aws.amazon.com/lambda/latest/dg/lambda-edge.html) and Amazon [CloudFront](https://aws.amazon.com/cloudfront/). For more information, see [Adding HTTP Security Headers Using Lambda@Edge and Amazon CloudFront](https://aws.amazon.com/blogs/networking-and-content-delivery/adding-http-security-headers-using-lambdaedge-and-amazon-cloudfront/).
+
+Include the following in your policy to allow the Amazon Chime SDK access to the meeting, messaging, and Amazon Voice Focus resources:
+
+```
+Content-Security-Policy: connect-src 'self' '*.chime.aws' 'wss://*.chime.aws' '*.amazonaws.com' 'https://*.sdkassets.chime.aws'; script-src 'https://*.sdkassets.chime.aws'; script-src-elem 'https://*.sdkassets.chime.aws'; worker-src 'blob:'; child-src 'blob:'
+```
+
+Note that the `script-src-elem` is not supported in Safari and Firefox. `worker-src` is not supported in Safari.
+
+You will need to add your own entries to this policy to allow for making connections and downloading scripts and assets required by your own application.
+
+## Meeting
+
+If you use audio, video, or screen sharing in your application by creating a [meeting session](https://github.com/aws/amazon-chime-sdk-js#meeting-session), ensure that your policy includes the following:
+
+`connect-src`: `*.chime.aws` `wss://*.chime.aws`
+
```

Comment:
This should be a block (triple backticks).

Reply:
Will change.